test/test_token_classificication_explainer.py

```diff
@@ -0,0 +1,238 @@
+import pytest
+from transformers import AutoModelForTokenClassification, AutoTokenizer
+from transformers_interpret import TokenClassificationExplainer
+from transformers_interpret.errors import (
+    AttributionTypeNotSupportedError,
+    InputIdsNotCalculatedError,
+)
+
+BERT_MODEL = AutoModelForTokenClassification.from_pretrained("dslim/bert-base-NER")
+BERT_TOKENIZER = AutoTokenizer.from_pretrained("dslim/bert-base-NER")
```

Comment:
For speed of testing it might be worth looking into if there are mini or distilbert versions of this model.